ipatests/test_ipaplatform/test_tasks.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2017  FreeIPA Contributors see COPYING for license
+#
+from __future__ import absolute_import
+
+import os
+
+from ipaplatform.tasks import tasks
+
+
+def test_ipa_version():
```

Comment:
This test is not needed in the backport, it has no relationship with the fix. I think you can simply remove it.

Reply:
Sure!